tests/Unit/Service/AvailableCarServiceTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Unit\Service;
+
+use App\Entity\Booking;
+use App\Entity\Car;
+use App\Entity\Station;
+use App\Service\AvailableCarService;
+use App\Service\PricingApi\PricingClientInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Persistence\ObjectRepository;
+use Doctrine\ORM\QueryBuilder;
+use PHPUnit\Framework\TestCase;
+
+class AvailableCarServiceTest extends TestCase
+{
+    public function test_returns_only_available_cars(): void
```

Comment:
```suggestion
    public function test_successful(): void
```
Non descriptive test name